container/compose.yml

```diff
@@ -80,6 +80,8 @@ services:
       KARAPACE_COMPATIBILITY: FULL
       KARAPACE_STATSD_HOST: statsd-exporter
       KARAPACE_STATSD_PORT: 8125
+      KARAPACE_KAFKA_SCHEMA_READER_STRICT_MODE: false
+      KARAPACE_KAFKA_RETRIABLE_ERRORS_SILENCED: true
```

Comment:
n00b question, why is this true by default?

Reply:
not at all, it's a valid question. So this doesn't affect the business logic of the service, so we enable it by default. This will cause us to reduce the alerts on our issue tracking (i.e. sentry) and as these errors are retried, it fine to silence them if the user wants, once again, they can opt out from this.